app/views/govuk_publishing_components/components/docs/big_number.yml

```diff
@@ -0,0 +1,75 @@
+name: Big number
+description: A big number, with a label if needed, for visualising quantitative values such as the number of government departments or historical prime ministers.
+body: |
+  This component needs at least one item in the `items` array passed to it, otherwise it won't render. If no `number` attribute is passed in the item, it will default to "0".
+accessibility_criteria: |
+  - uses clear language and is positioned in a logical context on the page it's being used on
```

Comment:
Just a small readability nitpick - I think this list needs to start with something like `The component must`.